<!DOCTYPE HTML>
<!--	
	This website has been built using Tessellate by HTML5 UP
	with some popup code from Dimension by HTML5 UP
	html5up.net | @ajlkn
	Free for personal and commercial use under the CCA 3.0 license (html5up.net/license)
-->
<html>
<head>
	<title>Computational Systems and Synthetic Biology</title>
	<meta charset="utf-8" />
	<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
	<link rel="stylesheet" href="assets/css/main.css" />
</head>
<body class="is-preload">

<!-- Header -->
<section id="header" class="dark">
	<header>
		<h1>Computational Systems and Synthetic Biology</h1>
		<p>Research group at University College London</p>
	</header>
	<footer>
		<a href="#projects" class="button scrolly">What do we do?</a>
	</footer>
</section>

<!-- PROJECTS -->
<section id="projects" class="main">
	<!-- PROJECTS - intro -->
	<header>
		<div class="container">
			<h2>quantitative biology + synthetic biology + systems biology</h2>
		</div>
	</header>
	<!-- PROJECTS - details -->
	<div class="content dark style3 featured">
		<div class="container">
			<!--- PROJECT - cartoons ----->
			<div class="row">
				<div class="col-3 col-12-narrow">
					<section>
						<span class="feature-icon"><a href="#cancer_popup" class="icon solid fa-dna"></a></span>
						<header>
							<h3>Mutational processes</h3>
						</header>
					</section>
				</div>
				<div class="col-3 col-12-narrow">
					<section>
						<span class="feature-icon"><a href="#microbiome_popup" class="icon solid fa-microscope"></a></span>
						<header>
							<h3>Microbiome engineering</h3>
						</header>
					</section>
				</div>
				<div class="col-3 col-12-narrow">
					<section>
						<span class="feature-icon"><a href="#grn_popup" class="icon solid fa-share-alt"></a></span>
						<header>
							<h3>Gene regulatory networks</h3>
						</header>
					</section>
				</div>
				<div class="col-3 col-12-narrow">
					<section>
						<span class="feature-icon"><a href="#biocomp_popup" class="icon solid fa-microchip"></a></span>
						<header>
							<h3>Biological computing</h3>
						</header>
					</section>
				</div>
				<div class="col-12">
					<footer>
						<a href="#people" class="button scrolly">Who are we?</a>
					</footer>
				</div>
			</div>
			<!--- PROJECT - popups ----->
			<!--- THIS IS ONE POPUP BLOCK ----->
			<a href="#x" class="overlay" id="cancer_popup"></a> <!---  ----->
			<div class="popup">
				<h2>Mutational processes</h2> <!---  ----->
				<p>We focus on the mechanistic modelling of genetic and sequence data to try to infer the biological
					processes that shape genomes. We have developed evolutionary models that can be fit to cancer
					patient sequencing data to infer important parameters such as mutation rates and selection
					coefficients. We are currently working on modelling structural variation and chromosome instability
					to try to understand how these important processes contribute to intra-tumour heterogeneity.
				</p>
				<footer>
					<a href="#research_cancer" class="button">Read more</a>
				</footer>
				<a class="close" href="#close"></a>
			</div>
			<!--- END BLOCK ----->

			<a href="#x" class="overlay" id="microbiome_popup"></a>
			<div class="popup">
				<h2>Microbiome engineering</h2>
				<p>The human intestine and the wide range of prokaryotic and eukaryotic organisms it supports form a
					mutualistic host-microbe symbiotic system crucial for many processes including the breakdown of
					plant polysaccharides and microbial fermentation. Variation and disruption of this natural
					ecosystem is linked to a diverse array of disorders including infectious disease, autoimmunity,
					obesity and cancer. We are engineering probiotic strains for therapeutic and sensing applications,
					plus understanding how to engineer microbial consortia.
				</p>
				<footer>
					<a href="#research_microbiome" class="button">Read more</a>
				</footer>
				<a class="close" href="#close"></a>
			</div>
			<!--- END BLOCK ----->

			<a href="#x" class="overlay" id="grn_popup"></a>
			<div class="popup">
				<h2>Gene regulatory networks</h2>
				<p>Using model space exploration from Bayesian statistics we have developed and parameterised a number
					of models of morphogenesis in collaboration with developmental biologists. We have also used the
					same methods to examine the robustness of simple gene networks in the context of synthetic biology,
					including the dynamics of multi-functional networks.
				</p>
				<footer>
					<a href="#research_grn" class="button">Read more</a>
				</footer>
				<a class="close" href="#close"></a>
			</div>
			<!--- END BLOCK ----->

			<a href="#x" class="overlay" id="biocomp_popup"></a>
			<div class="popup">
				<h2>Biological computing</h2>
				<p>Biological organisms comprise complex information processing systems and computation is present at
					every level, from protein and DNA nanomachines, through cellular gene and signalling networks to
					tissues, brains and ecosystems. Thus, computation can be seen to be a fundamental and unifying
					theme of biology. This computation is performed in a highly parallel and robust fashion, using
					concurrent, multi-scale hybrid digital and analogue processing, often utilising components that are
					unreliable and noisy, and at orders of magnitude lower power requirements than traditional,
					“silicon” computing. Understanding how biological computing is performed thus has enormous
					potential for driving forward biology and engineering, enabling a more sustainable future in an
					increasingly computation-dependent world.
				</p>
				<footer>
					<a href="#research_biocomp" class="button">Read more</a>
				</footer>
				<a class="close" href="#close"></a>
			</div>
			<!--- END BLOCK ----->
		</div>
	</div>
</section>

<!-- PEOPLE -->
<section id="people" class="main">
	<!-- PEOPLE - intro -->
	<header>
		<div class="container">
			<h2>Interdisciplinary and collaborative</h2>
			<p>We have backgrounds ranging across physics, computer science, microbiology, synthetic biology and genetics.</p>
			<p>
				<figure>
					<img width="50%" src="images/group/barnes-group-coal-drops-yard-2021.jpg" alt="" />
					<figcaption> Coal Drops yard, 2021 </figcaption>
				</figure>
				<figure>
					<img width="50%" src="images/group/barnes-group-regents-2020.jpg" alt="" />
					<figcaption> Regents park, 2020 </figcaption>
				</figure>
				<figure>
					<img width="50%" src="images/group/barnes-group-xmas-2019.jpg" alt="" />
					<figcaption> Rockefeller Building xmas party, 2019 </figcaption>
				</figure>
				<figure>
					<img width="50%" src="images/group/barnes-group-bath-2018.jpg" alt="" /> <br>
					<figcaption> Group retreat, Bath 2018 </figcaption>
				</figure>
			</p>
			<!-- <img src="images/people/barnes-group-bath.jpg" alt="" /> -->
		</div>
	</header>
	<!-- PEOPLE - details -->
	<div class="content dark style2">
		<div class="container">
			<!--- PEOPLE - photos ----->
			<h1><b>Current group members</b></h1>
			<div class="row">
				<div class="col-12 col-12-narrow">
					<div class="row">
						<div class="col-3"><a href="#Chris_Barnes" class="image fit"><img src="images/people/photo-barnes.jpg" alt="" /></a>
							Prof. Chris P Barnes
						</div>
						<div class="col-3"><a href="#Linda_Dekker" class="image fit"><img src="images/people/photo-dekker.jpg" alt="" /></a>
							Dr Linda Dekker
						</div>
						<div class="col-3"><a href="#Alex_Fedorec" class="image fit"><img src="images/people/photo-fedorec.jpg" alt="" /></a>
							Dr Alex Fedorec
						</div>
						<div class="col-3"><a href="#Bingxin_Lu" class="image fit"><img src="images/people/photo-lu.png" alt="" /></a>
							Dr Bingxin Lu
						</div>
						<div class="col-3"><a href="#Ke_Yan_Wen" class="image fit"><img src="images/people/photo-wen.jpg" alt="" /></a>
							Dr Ke Yan Wen
						</div>
						<div class="col-3"><a href="#Mohamed_Ali_Al_Badri" class="image fit"><img src="images/people/photo-al-badri.jpg" alt="" /></a>
							Dr Mohamed Ali Al-Badri
						</div>
						<div class="col-3"><a href="#Jack_Rutter" class="image fit"><img src="images/people/photo-rutter2.jpg" alt="" /></a>
							Dr Jack Rutter
						</div>
						<div class="col-3"><a href="#Neythen_Treloar" class="image fit"><img src="images/people/photo-treloar.png" alt="" /></a>
							Neythen Treloar
						</div>
						<div class="col-3"><a href="#Kim_Owen" class="image fit"><img src="images/people/photo-owen.jpg" alt="" /></a>
							Kimberley Owen
						</div>
						<div class="col-3"><a href="#Jurgen_Riedel" class="image fit"><img src="images/people/photo-riedel.jpg" alt="" /></a>
                                                        Jurgen Riedel
                                                </div>
					</div>
				</div>
			</div>
			<!--- PEOPLE - popups ----->
			<!--- THIS IS ONE POPUP BLOCK ----->
			<a href="#x" class="overlay" id="Chris_Barnes"></a>
			<div class="popup">
				<h2>Prof. Chris Barnes<br>christopher.barnes@ucl.ac.uk</h2>
				<p>I'm a Professor in the Division of Biosciences within the Faculty of Life Sciences at
					University College London. My career has spanned multiple fields including particle
					physics, genetics, statistics, computational systems biology and synthetic biology (<a href=https://scholar.google.co.uk/citations?user=aVCoSW8AAAAJ&hl=en>Google Scholar</a>). In
					addition to my research, I'm also actively engaged in training the next generation of
					researchers; I developed the BBSRC funded e-learning resource <a href=http://www.sysmic.ac.uk> SysMIC </a>
					to train life scientists in computational and mathematical skills. I'm also involved in running the
					EPSRC funded Centre for Doctoral Training in BioDesign Engineering and the LIDo Doctoral Training Partnership.</p>
				<a class="close" href="#close"></a>
			</div>
			<!--- END BLOCK ----->
			<a href="#x" class="overlay" id="Linda_Dekker"></a>
			<div class="popup">
				<h2>Dr Linda Dekker<br>linda.dekker.09@ucl.ac.uk</h2>
				<p>I am currently working on a project constructing, characterising and optimising
					whole-cell bacterial biosensors that detect metabolites important in gut microbiota
					disorders. I completed my BSc and MSc in Microbiology at the University of Otago
					(New Zealand) and did a PhD in Microbiology in Prof Joanne Santini's group at UCL. I
					previously worked as a PDRA at Imperial College London on various synthetic biology
					projects ranging from natural product biosynthesis in Clostridia, to changing the material
					properties of bacterial cellulose.  I'm passionate about Microbiology and am a Microbiology
					Society Champion and try to promote the awesome opportunities the
					<a href=https://www.microbiologyresearch.org/>Microbiology Society</a> has to offer. I
					founded the Barnes Lab cake club &#x1F9C1;.
				</p>
				<a class="close" href="#close"></a>
			</div>
			<a href="#x" class="overlay" id="Alex_Fedorec"></a>
			<div class="popup">
				<h2>Dr Alex Fedorec<br>alexander.fedorec.13@ucl.ac.uk</h2>
				<p>I am a Postdoctoral Research Associate at University College London in the Division of
					Biosciences. My current research interests are in the design of synthetic microbial
					communities and their application in healthcare and industrial biotechnology. I completed
					my PhD in Synthetic Biology in Chris Barnes' group and did my Masters degree in Modelling
					Biological Complexity through <a href=https://www.ucl.ac.uk/complex/>CoMPLEX</a> at UCL. My
					Bachelor's degree in Computer Science was undertaken at the University of Bath, where I
					developed an interest, through working with Dr Joanna Bryson, in using models to understand
					Natural Intelligence. I co-founded and currently co-organise the
					<a href=https://londonsynbionetwork.wordpress.com/>London SynBio Network</a>. I previously
					worked in industry as a software engineer for Research In Motion (now BlackBerry).</p>
				<a class="close" href="#close"></a>
			</div>
			<a href="#x" class="overlay" id="Bingxin_Lu"></a>
			<div class="popup">
				<h2>Dr Bingxin Lu<br>b.lu@ucl.ac.uk</h2>
				<p>I am a Research Fellow in Chris Barnes’s group at Department of Cell & Developmental
					Biology University College London. My general research interest is using computational
					approaches to study biological problems. Currently, I mainly work on modelling of
					chromosomal instability in cancer evolution. Previously, I was a Postdoctoral Fellow at
					Genome Institute of Singapore (Weiwei Zhai’s Group), where I worked on tumour heterogeneity
					and clonal evolution. I completed my PhD in Computational Biology under the supervision of
					Hon Wai Leong at School of Computing National University of Singapore, where I developed
					computational methods for analyzing lateral gene transfer. I obtained my Master's and
					Bachelor's degree at Software Engineering Institute East China Normal University, where I
					developed data analysis platforms for RNA-Seq and proteomics data.
				</p>
				<a class="close" href="#close"></a>
			</div>
			<a href="#x" class="overlay" id="Ke_Yan_Wen"></a>
			<div class="popup">
				<h2>Dr Ke Yan Wen<br>k.wen@ucl.ac.uk</h2>
				<p>I joined the Barnes group as a Postdoctoral Research Associate in October 2018 to work
					on spatial computing using bacterial patterning. Previously I was a postdoc in Associate
					Prof. Wen Shan Yew’s group at the National University of Singapore, where I focused on
					enzyme library screening using cell-free expression systems. I completed my PhD in
					Synthetic Biology with Prof. Paul Freemont at Imperial College London, developing cell-free
					biosensors for monitoring cystic fibrosis lung infections. I’m interested in creating
					robust diagnostics for medical applications, lab automation, and (most recently) tinkering
					with genetic circuits.
				</p>
				<a class="close" href="#close"></a>
			</div>
			<a href="#x" class="overlay" id="Mohamed_Ali_Al_Badri"></a>
			<div class="popup">
				<h2>Dr Mohamed Ali Al-Badri<br>m.al-badri@ucl.ac.uk</h2>
				<p>I am a Research Fellow in Statistical & Systems Biology in the Division of Biosciences at UCL. My
					work involves using Deep Learning methods to study structural variation in cancer. I completed my
					PhD with the London Interdisciplinary Doctoral Programme (LIDo) at King's College London, where I
					used techniques in computational theoretical physics to simulate biological systems and the
					bio-nano interface. During the PhD, I did an internship at The Alan Turing Institute, where I was
					part of the Uncertainty Quantification of Multi-scale and Multi-physics computer Models (UQMMM)
					project. I have an undergraduate degree in physics with theoretical physics from King's College
					London.
				</p>
				<a class="close" href="#close"></a>
			</div>
			<a href="#x" class="overlay" id="Jack_Rutter"></a>
			<div class="popup">
				<h2>Dr Jack Rutter<br>jack.rutter.16@ucl.ac.uk</h2>
				<p>I am currently a Post Doctoral Research Associate, at University College London in the CSSB lab. 
				   I previously completed my PhD, supervised by Prof. Chris Barnes and Prof. Geraint Thomas, where I developed a range of novel whole-cell biosensors. 
				   I now work on microbiome engineering. My current project is looking at how we can engineer probiotic E. coli to target and remove pathogenic species of bacteria in the body.
				</p>
				<a class="close" href="#close"></a>
			</div>
			<a href="#x" class="overlay" id="Neythen_Treloar"></a>
			<div class="popup">
				<h2>Neythen Treloar<br>neythen.treloar.14@ucl.ac.uk</h2>
				<p>I am a PhD student at University College London in the Division of Biosciences. My
					Bachelor's degree in Natural Sciences was undertaken at UCL, I specialised in Physics and
					Molecular and Cell Biology. This is where I first became interested in applying
					quantitative techniques to Synthetic Biology. I also completed my Masters degree in
					Scientific Computing at UCL, during which I used machine learning to optimise bacterial
					communities. My current research interests are in the use of mathematical and computational
					tools to design synthetic biological systems, we are currently working on the construction
					of a spatial biological computer and the optimisation of bacterial communities. Some of my
					other work includes the design of optimal experiments using machine learning.
				</p>
				<a class="close" href="#close"></a>
			</div>
			<a href="#x" class="overlay" id="Kim_Owen"></a>
			<div class="popup">
				<h2>Kimberley Owen<br>k.owen20@imperial.ac.uk</h2>
				<p>I am a PhD student on the CDT for Biodesign
					Engineering. I am new to synthetic biology and
					looking forward to expanding my skill set. I
					spent the summer of 2020 working at the Milton
					Keynes Lighthouse Lab (UKBiocentre) testing
					covid-19 samples. I completed my Bachelor’s in
					Biochemistry at the University of Bath (2020)
					and conducted a year in research (2018/19) at
					the University of Arizona (Herbst-Kralovetz
					group) where I was introduced to the complex
					interplay between host and microbiota in the
					female reproductive tract. My PhD project will
					involve developing new bacterial therapeutics
					for targeting and manipulating the cancer microbiome.
				</p>
				<a class="close" href="#close"></a>
			</div>
			<a href="#x" class="overlay" id="Jurgen_Riedel"></a>
                        <div class="popup">
                                <h2>Jurgen Riedel<br>jurgen.riedel@gmail.com</h2>
                                <p>I am a PhD student at University College London in the Department of Cell and Developmental Biology. 
				   I've had a career as a Data Analyst and Scientist in many industries, including private banking, energy and biotechnology. 
				   I hold a PhD in Theoretical Physics from the University of Oldenburg, Germany, where my research focused on black holes, quantum gravity and dark matter. 
				   I'm also deeply interested in complexity theory and its application to large universe masses, including nature at the cellular level. 
				   My current research focuses on reaction-diffusion systems in the continuous domain via PDEs and the discrete domain via cellular automata. 
				   My goal is to study complex systems at the edge of chaos and observe the emergence of complex behaviour.
                                </p>
                                <a class="close" href="#close"></a>
                        </div>
		</div>
	</div>
	<div class="content dark style1">
		<div class="container">
			<h1> <b>Past group members</b></h1>
			Dr Behzad Karkaria. PhD student 2017-2021. Now interning at Hummingbird Diagnostics<br>
			Dr Luca Rosa. PhD student 2017-2021. Now on placement at CC Bio<br>
			Clare Robinson. Research Assistant 2019-2020. Now doing a PhD with David Riglar, Imperial College  <br>
			Dr Josh Russell-Buckland. PhD student 2016-2020. Now a software engineer at Guardian News and Media <br>
			Dr Marc Williams. PhD student 2014-2018. Now a PDRA with Sohrab Shah, Memorial Sloan Kettering <br>
			Dr Tanel Ozdemir. PhD student 2013-2017. Now an Investment Associate at AlbionVC <br>
			Dr Lourdes Sriraja. PhD student 2013-2017. Now a PDRA with Evangelia Petsalaki, EBI <br>
			Dr Mae Woods. PDRA 2013-2018. Now at Baylor College of Medicine  <br>
			Dr Miriam Leon. PhD student 2012-2016. Now a data scientist at Stitch Fix <br>
			<br>
		</div>
	</div>
</section>

<section id="research_cancer" class="main">
	<header>
		<div class="container">
			<h2>Mutational processes</h2>
			<figure>
				<img width="50%" src="images/science/PDTO-model.jpg" alt="" />
				<figcaption>Fig: Modelling chromosomal gain and loss within a stochastic branching process </figcaption>
			</figure>
			<p> Bollen, Y., Stelloo, E., van Leenen, P., van den Bos, M., Ponsioen, B., Lu, B., ...Snippert H.J.G. (2021). 
			  Reconstructing single-cell karyotype alterations in colorectal cancer identifies punctuated and gradual diversification patterns. 
			  Nature Genetics, doi:<a href=https://www.nature.com/articles/s41588-021-00891-2>10.1038/s41588-021-00891-2</a>
			  </p>
			<p> Williams, M.J., Zapata, L., Werner, B., Barnes, C.P., Sottoriva, A., Graham, T.A. (2020).
				Measuring the distribution of fitness effects in somatic evolution by combining clonal dynamics with dN/dS ratios.
				eLife, 9 doi:<a href=https://doi.org/10.7554/eLife.48714>10.7554/eLife.48714</a>
			</p>
			<p>Williams, M.J., Werner, B., Heide, T., Curtis, C., Barnes, C.P., Sottoriva, A., Graham, T.A. (2018).
				Quantification of subclonal selection in cancer from bulk sequencing data.
				Nature Genetics 2018 Jun;50(6):895-903. doi:<a href=https://doi.org/10.1038/s41588-018-0128-6>10.1038/s41588-018-0128-6</a> <br>
				<a href=https://www.thetimes.co.uk/edition/news/tumour-growth-can-be-predicted-by-computer-bdqp5c6ww> Article in The Times </a>
			</p>
			<p>
				Woods M.L., Barnes C.P. (2016).
				Mechanistic Modelling and Bayesian Inference Elucidates the Variable Dynamics of Double-Strand Break Repair.
				PLoS Comput Biol. 2016 Oct 14;12(10):e1005131. doi:<a href=https://doi.org/10.1371/journal.pcbi.1005131>10.1371/journal.pcbi.1005131</a>
			</p>
			<p>
				Williams, M.J., Werner, B, Barnes, C.P., Graham, T.A., & Sottoriva, A. (2016).
				Identification of neutral tumor evolution across cancer types.
				Nature Genetics, 2016 Jan 18. doi:<a href=https://doi.org/10.1038/ng.3489>10.1038/ng.3489</a>
			</p>
			<p>
				Chaidos, A, Barnes, C.P. et. al. (2013).
				Clinical drug resistance linked to interconvertible phenotypic and functional states of tumor-propagating cells in multiple myeloma.
				Blood. 2013 Jan 10;121(2):318-28. doi:<a href=https://doi.org/10.1182/blood-2012-06-436220>10.1182/blood-2012-06-436220</a>
			</p>
		</div>
	</header>
</section>

<section id="research_microbiome" class="main">
	<header>
		<div class="container">
			<h2>Microbiome engineering</h2>
			<figure>
				<img width="40%" src="images/science/photo-worm-v1-1200x800.jpg" alt="" />
				<img width="35%" src="images/science/spock-grn.jpg" alt="" />
				<figcaption>
					Fig: (Left) Fluorescent bacteria colonising the gut of a C. elegans nematode worm.
					(Right)	An engineered GRN in E. coli enabling bacteriocin production control through quorum sensing (SPoCK system)
				</figcaption>
			</figure>
			<p>
			  Rutter, J.W., Dekker, L., Fedorec, A.J.H., Gonzales, D.T., Wen, K.Y., Tanner, L.E.S., ...Barnes, C.P. (2021). 
			  Engineered acetoacetate-inducible whole-cell biosensors based on the AtoSC two-component system.
			  Biotechnology and Bioengineering, doi:<a href=https://onlinelibrary.wiley.com/doi/10.1002/bit.27897>10.1002/bit.27897</a><br>
			  <a href=https://zenodo.org/record/5078479#.YRp_JGbdvUI> Download the data here </a> <br>
			  </p>
			<p>
				Fedorec, A.J.H., Karkaria, B.D., Sulu, M., Barnes, C.P. (2021).
				Single strain control of microbial consortia.
				Nat Commun, 12, 1977. doi:<a href=https://www.nature.com/articles/s41467-021-22240-x>10.1038/s41467-021-22240-x</a><br>
				<a href=https://zenodo.org/record/4560355#.YRqAT2bdvUI> Download the data here </a> <br>
			</p>
			<p>
				Karkaria, B.D., Fedorec, A.J.H., Barnes, C.P. (2021).
				Automated design of synthetic microbial communities.
				Nat Commun, 12, 672. doi:<a href=https://www.nature.com/articles/s41467-020-20756-2>10.1038/s41467-020-20756-2</a><br>
				<a href=https://doi.org/10.5281/zenodo.4286040> Download the data here </a> <br>
				<a href=https://github.com/ucl-cssb/AutoCD>Code available on GitHub</a>
			</p>
			<p>
				Fedorec, A.J.H., Robinson, C.M., Wen, K.Y., Barnes, C.P. (2020).
				FlopR: An Open Source Software Package for Calibration and Normalization of Plate Reader and FlowCytometry Data.
				ACS Synth Biol, <a href=https://pubs.acs.org/doi/10.1021/acssynbio.0c00296>doi:10.1021/acssynbio.0c00296</a><br>
				<a href=https://doi.org/10.5281/zenodo.3977408> Download the data here </a><br>
				<a href=https://github.com/ucl-cssb/flopr>Code available on GitHub</a>
			</p>
			<p>
				Treloar, N.J., Fedorec, A.J.H., Ingalls, B., Barnes, C.P. (2020).
				Deep reinforcement learning for the control of microbial co-cultures in bioreactors.
				PLoS Comput Biol, 16 (4), e1007783. <a href=https://doi.org/doi:10.1371/journal.pcbi.1007783>doi:10.1371/journal.pcbi.1007783</a><br>
				<a href=https://zenodo.org/record/3728079#.XyAF5_jdtTY> Download the data here </a> <br>
				<a href=https://github.com/ucl-cssb/ROCC>Code available on GitHub</a>
			</p>
			<p>
				Wen, K.Y., Rutter, J.W., Barnes, C.P., Dekker, L. (2019). Fundamental Building Blocks of Whole-Cell Biosensor Design.
				In Thouand, G. (Ed.), Handbook of Cell Biosensors. (pp. 1-23). Springer International Publishing.
			</p>
			<p>
				Rutter, J.W., Ozdemir, T., Galimov, E.R., Quintaneiro, L.M., Rosa, L., Thomas, G.M., Cabreiro, F., Barnes, C.P. (2019).
				Detecting Changes in the Caenorhabditis elegans Intestinal Environment Using an Engineered Bacterial Biosensor.
				ACS Synth Biol, doi:<a href=https://doi.org/10.1021/acssynbio.9b00166>10.1021/acssynbio.9b00166 </a> <br>
				<a href=https://zenodo.org/record/3629222#.XlEPH5OgJR5> Download the data and code here </a>
			</p>
			<p>
				Shaw, L., Bassam, H., Barnes, C.P., Walker, A., Klein, N., Balloux, F. (2019).
				Modelling microbiome recovery after antibiotics using a stability landscape framework.
				ISME Journal, doi:<a href=https://doi.org/10.1038/s41396-019-0392-1>10.1038/s41396-019-0392-1 </a> <br>
				<a href=https://www.telegraph.co.uk/science/2019/03/22/single-course-antibiotics-may-cause-irreversible-damage-crucial/> Article in The Telegraph </a>
			</p>
			<p>
				Fedorec, A.J.H., Ozdemir, T., Doshi, A., Ho, Y.-.K., Rosa, L., Rutter, J., ...Barnes, C.P. (2019).
				Two New Plasmid Post-segregational Killing Mechanisms for the Implementation of Synthetic Gene Networks in Escherichia coli.
				iScience, doi:<a href=https://doi.org/10.1016/j.isci.2019.03.019>10.1016/j.isci.2019.03.019 </a> <br>
				<a href=https://data.mendeley.com/datasets/6kggw28x8m/1> Download the data and code here </a> <br>
				<a href=https://www.addgene.org/browse/article/28206875/> Plasmids available on addgene </a>

			<p> Barnes, C.P., Fedorec, A.J.H. (2018). Five amazing ways redesigning biological cells could help us fight cancer.
				<a href=https://theconversation.com/five-amazing-ways-redesigning-biological-cells-could-help-us-fight-cancer-95256> The Conversation </a>
			</p>
			<p>
				Ozdemir, T., Fedorec, A.J.H., Danino, T., Barnes, C.P. (2018).
				Synthetic Biology and Engineered Live Biotherapeutics: Toward Increasing System Complexity.
				Cell Systems, 7 (1), 5-16. doi:<a href=https://doi.org/10.1016/j.cels.2018.06.008>10.1016/j.cels.2018.06.008 </a>
			</p>
		</div>
	</header>
</section>

<section id="research_grn" class="main">
	<header>
		<div class="container">
			<h2>Gene regulatory networks</h2>
			<figure>
				<img width="50%" src="images/science/fig10-toggle-osc-v2.jpg" alt="" />
				<figcaption>Fig: Post-translational coupling of an oscillator and a bistable switch</figcaption>
			</figure>
			<p>Perez-Carrasco, R., Barnes, C.P., Schaerli, Y., Isalan, M., Briscoe, J., Page, K.M. (2018).
				Combining a Toggle Switch and a Repressilator within the AC-DC Circuit Generates Distinct Dynamical Behaviors.
				Cell Systems. doi:<a href=https://doi.org/10.1016/j.cels.2018.02.008>10.1016/j.cels.2018.02.008</a>
			</p>
			<p>
				Boeing, P., Leon, M., Nesbeth, D.N., Finkelstein, A., Barnes, C. (2018).
				Towards an Aspect-Oriented Design and Modelling Framework for Synthetic Biology.
				Processes, 6 (9), doi:<a href=https://doi.org/10.3390/pr6090167>10.3390/pr6090167</a>
			</p>
			<p>
				Leon, M., Woods, M.L., Fedorec, A.J.H., & Barnes, C.P. (2016)
				A computational method for the investigation of multistable systems and its application to genetic switches.
				BMC Systems Biology 2016 Dec 7;10(1):130. doi:<a href=https://doi.org/10.1186/s12918-016-0375-z>10.1186/s12918-016-0375-z</a>
			</p>
			<p>
				Woods, M., Leon, M., Perez-Carrasco, R. & Barnes, C.P. (2016)
				A statistical approach reveals designs for the most robust stochastic gene oscillators.
				ACS Synthetic Biology 5 (6), pp 459–470. doi:<a href=https://doi.org/10.1021/acssynbio.5b00179>10.1021/acssynbio.5b00179</a>
			</p>
			<p>
				Cohen, M., Kicheva, A., Ribeiro, A., Blassberg, R., Page, K. M., Barnes, C. P. & Briscoe, J. (2015).
				Ptch1 and Gli regulate Shh signalling dynamics via multiple mechanisms.
				Nature Communications, 6. doi:<a href=https://doi.org/10.1038/ncomms7709>10.1038/ncomms7709</a>
			</p>
			<p>
				Cohen, M., Page, K. M., Perez-Carrasco, R., Barnes, C. P., & Briscoe, J. (2014).
				A theoretical framework for the regulation of Shh morphogen-controlled gene expression.
				Development, 141(20), 3868-3878. doi:<a href=https://doi.org/10.1242/dev.112573>10.1242/dev.112573</a>
			</p>
			<p>
				Liepe, J., Kirk, P., Filippi, S., Toni, T., Barnes, C. P., & Stumpf, M. P. H. (2014).
				A framework for parameter estimation and model selection from experimental data in systems biology using approximate Bayesian computation.
				Nature Protocols, 9(2), 439-456. doi:<a href=https://doi.org/10.1038/nprot.2014.025>10.1038/nprot.2014.025</a>
			</p>
		</div>
	</header>
</section>

<section id="research_biocomp" class="main">
	<header>
		<div class="container">
			<h2>Biological computing</h2>
			<figure>
				<img width="50%" src="images/science/logic_gate_sim.jpg" alt="" />
				<figcaption>Fig: Patterning as a spatial computation. Each panel represents a different interpretation
					of a morphogen gradient produced from two sources A and B</figcaption>
			</figure>
<p>Treloar, N., Wen, K.Y., Fedorec, A., Barnes, C. (2021). SynBioBrain: building biological computers from bacterial populations. The Project Repository Journal, doi:<a href=https://doi.org/10.54050/PRJ1117751>doi.org/10.54050/PRJ1117751</a> </p>

			<p> Karkaria, B.D., Treloar, N.J., Barnes, C.P., Fedorec, A.J.H. (2020).
				From microbial communities to distributed computing systems
				Frontiers in Bioengineering and Biotechnology 8, 834, doi:<a href=https://doi.org/10.3389/fbioe.2020.00834>doi.org/10.3389/fbioe.2020.00834</a>
			</p>
			<p>Dalchau, N., Szép, G., Hernansaiz-Ballesteros, R., Barnes, C.P., Cardelli, L., Phillips, A., Csikász-Nagy, A. (2018).
				Computing with biological switches and clocks.
				Natural Computing, 1-19. doi:<a href=https://doi.org/10.1007/s11047-018-9686-x>10.1007/s11047-018-9686-x</a>
			</p>
		</div>
	</header>
</section>

<!-- Footer -->
<section id="footer">
	<ul class="icons">
		<li><a href="https://twitter.com/cssb_lab" class="icon brands fa-twitter"><span class="label">Twitter</span></a></li>
		<li><a href="https://github.com/ucl-cssb" class="icon brands fa-github"><span class="label">GitHub</span></a></li>
	</ul>
	<div class="copyright">
		<ul class="menu">
			<li>&copy; Untitled. All rights reserved.</li><li>Design: <a href="http://html5up.net">HTML5 UP</a></li>
		</ul>
	</div>
</section>

<!-- Scripts -->
<script src="assets/js/jquery.min.js"></script>
<script src="assets/js/jquery.scrolly.min.js"></script>
<script src="assets/js/browser.min.js"></script>
<script src="assets/js/breakpoints.min.js"></script>
<script src="assets/js/util.js"></script>
<script src="assets/js/main.js"></script>

</body>
</html>
