<!DOCTYPE HTML>
<!--	
	This website has been built using Tessellate by HTML5 UP
	with some popup code from Dimension by HTML5 UP
	html5up.net | @ajlkn
	Free for personal and commercial use under the CCA 3.0 license (html5up.net/license)
-->
<html>
<head>
	<title>Computational Systems and Synthetic Biology</title>
	<meta charset="utf-8" />
	<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
	<link rel="stylesheet" href="assets/css/main.css" />
</head>
<body class="is-preload">

<!-- Header -->
<section id="header" class="dark">
	<header>
		<h1>Computational Systems and Synthetic Biology</h1>
		<p>Research group at University College London</p>
	</header>
	<footer>
		<a href="#projects" class="button scrolly">What do we do?</a>
	</footer>
</section>

<!-- PROJECTS -->
<section id="projects" class="main">
	<!-- PROJECTS - intro -->
	<header>
		<div class="container">
			<h2>quantitative biology + synthetic biology + systems biology</h2>
		</div>
	</header>
	<!-- PROJECTS - details -->
	<div class="content dark style3 featured">
		<div class="container">
			<!--- PROJECT - cartoons ----->
			<div class="row">
				<div class="col-4 col-12-narrow">
					<section>
						<span class="feature-icon"><a href="#cancer_popup" class="icon solid fa-dna"></a></span>
						<header>
							<h3>Mutational processes</h3>
						</header>
					</section>
				</div>
				<div class="col-4 col-12-narrow">
					<section>
						<span class="feature-icon"><a href="#microbiome_popup" class="icon solid fa-microscope"></a></span>
						<header>
							<h3>Microbiome engineering</h3>
						</header>
					</section>
				</div>
				<!-- <div class="col-3 col-12-narrow">
					<section>
						<span class="feature-icon"><a href="#grn_popup" class="icon solid fa-share-alt"></a></span>
						<header>
							<h3>Gene regulatory networks</h3>
						</header>
					</section>
				</div> -->
				<div class="col-4 col-12-narrow">
					<section>
						<span class="feature-icon"><a href="#biocomp_popup" class="icon solid fa-microchip"></a></span>
						<header>
							<h3>Biological computation</h3>
						</header>
					</section>
				</div>
				<div class="col-12">
					<footer>
						<a href="#people" class="button scrolly">Who are we?</a>
					</footer>
				</div>
			</div>
			<!--- PROJECT - popups ----->
			<!--- THIS IS ONE POPUP BLOCK ----->
			<a href="#x" class="overlay" id="cancer_popup"></a> <!---  ----->
			<div class="popup">
				<h2>Mutational processes</h2> <!---  ----->
				<p>We focus on the mechanistic modelling of genetic and sequence data to try to infer the biological
					processes that shape genomes. We have developed evolutionary models that can be fit to cancer
					patient sequencing data to infer important parameters such as mutation rates and selection
					coefficients. We are currently working on modelling structural variation and chromosome instability
					to try to understand how these important processes contribute to intra-tumour heterogeneity.
				</p>
				<footer>
					<a href="#research_cancer" class="button">Read more</a>
				</footer>
				<a class="close" href="#close"></a>
			</div>
			<!--- END BLOCK ----->

			<a href="#x" class="overlay" id="microbiome_popup"></a>
			<div class="popup">
				<h2>Microbiome engineering</h2>
				<p>The human intestine and the wide range of prokaryotic and eukaryotic organisms it supports form a
					mutualistic host-microbe symbiotic system crucial for many processes including the breakdown of
					plant polysaccharides and microbial fermentation. Variation and disruption of this natural
					ecosystem is linked to a diverse array of disorders including infectious disease, autoimmunity,
					obesity and cancer. We are engineering probiotic strains for therapeutic and sensing applications,
					plus understanding how to engineer microbial consortia.
				</p>
				<footer>
					<a href="#research_microbiome" class="button">Read more</a>
				</footer>
				<a class="close" href="#close"></a>
			</div>
			<!--- END BLOCK ----->

			<a href="#x" class="overlay" id="grn_popup"></a>
			<div class="popup">
				<h2>Gene regulatory networks</h2>
				<p>Using model space exploration from Bayesian statistics we have developed and parameterised a number
					of models of morphogenesis in collaboration with developmental biologists. We have also used the
					same methods to examine the robustness of simple gene networks in the context of synthetic biology,
					including the dynamics of multi-functional networks.
				</p>
				<footer>
					<a href="#research_grn" class="button">Read more</a>
				</footer>
				<a class="close" href="#close"></a>
			</div>
			<!--- END BLOCK ----->

			<a href="#x" class="overlay" id="biocomp_popup"></a>
			<div class="popup">
				<h2>Biological computation</h2>
				<p>Biological organisms comprise complex information processing systems and computation is present at
					every level, from protein and DNA nanomachines, through cellular gene and signalling networks to
					tissues, brains and ecosystems. Thus, computation can be seen to be a fundamental and unifying
					theme of biology. This computation is performed in a highly parallel and robust fashion, using
					concurrent, multi-scale hybrid digital and analogue processing, often utilising components that are
					unreliable and noisy, and at orders of magnitude lower power requirements than traditional,
					“silicon” computing. Understanding how biological computing is performed thus has enormous
					potential for driving forward biology and engineering, enabling a more sustainable future in an
					increasingly computation-dependent world.
				</p>
				<footer>
					<a href="#research_biocomp" class="button">Read more</a>
				</footer>
				<a class="close" href="#close"></a>
			</div>
			<!--- END BLOCK ----->
		</div>
	</div>
</section>

<!-- PEOPLE -->
<section id="people" class="main">
	<!-- PEOPLE - intro -->
	<header>
		<div class="container">
			<h2>Interdisciplinary and collaborative</h2>
			<p>We have backgrounds ranging across physics, computer science, microbiology, synthetic biology and genetics.</p>
			<p>
				<figure>
					<img width="50%" src="images/group/barnes-group-stratford-2022.jpg" alt="" /> <br>
					<figcaption> Group retreat, Stratford 2022 </figcaption>
				</figure>
				<figure>
					<img width="50%" src="images/group/barnes-group-coal-drops-yard-2021.jpg" alt="" />
					<figcaption> Coal Drops yard, 2021 </figcaption>
				</figure>
				<figure>
					<img width="50%" src="images/group/barnes-group-regents-2020.jpg" alt="" />
					<figcaption> Regents park, 2020 </figcaption>
				</figure>
				<figure>
					<img width="50%" src="images/group/barnes-group-xmas-2019.jpg" alt="" />
					<figcaption> Rockefeller Building xmas party, 2019 </figcaption>
				</figure>
				<figure>
					<img width="50%" src="images/group/barnes-group-bath-2018.jpg" alt="" /> <br>
					<figcaption> Group retreat, Bath 2018 </figcaption>
				</figure>
			</p>
			<!-- <img src="images/people/barnes-group-bath.jpg" alt="" /> -->
		</div>
	</header>
	<!-- PEOPLE - details -->
	<div class="content dark style2">
		<div class="container">
			<!--- PEOPLE - photos ----->
			<h1><b>Current group members</b></h1>
			<div class="row">
				<div class="col-12 col-12-narrow">
					<div class="row">
						<div class="col-3"><a href="#Chris_Barnes" class="image fit"><img src="images/people/photo-barnes2.jpg" alt="" /></a>
							Prof. Chris P Barnes
						</div>
						<div class="col-3"><a href="#Linda_Dekker" class="image fit"><img src="images/people/photo-dekker.jpg" alt="" /></a>
							Dr Linda Dekker
						</div>
						<div class="col-3"><a href="#Pedro_Fontanarrosa" class="image fit"><img src="images/people/photo-fontanarrosa.jpg" alt="" /></a>
							Dr Pedro Fontanarrosa
						</div>
						<div class="col-3"><a href="#Soutrick_Das" class="image fit"><img src="images/people/photo-das.jpg" alt="" /></a>
							Dr Soutrick Das
						</div>
						<div class="col-3"><a href="#Jurgen_Riedel" class="image fit"><img src="images/people/photo-riedel.jpg" alt="" /></a>
                            Dr Jurgen Riedel
						</div>
						<div class="col-3"><a href="#Kim_Owen" class="image fit"><img src="images/people/photo-owen.jpg" alt="" /></a>
							Kimberley Owen
						</div>
						<div class="col-3"><a href="#Chania_Clare" class="image fit"><img src="images/people/photo-clare.jpg" alt="" /></a>
                            Chania Clare
						</div>
						<div class="col-3"><a href="#Kathleen_Zhang" class="image fit"><img src="images/people/photo-zhang.jpg" alt="" /></a>
                            Kathleen Zhang
						</div>
						<div class="col-3"><a href="#Casey_Chen" class="image fit"><img src="images/people/photo-chen.jpeg" alt="" /></a>
                            Casey Chen
						</div>
					</div>
				</div>
			</div>
			<!--- PEOPLE - popups ----->
			<!--- THIS IS ONE POPUP BLOCK ----->
			<a href="#x" class="overlay" id="Chris_Barnes"></a>
			<div class="popup">
				<h2>Prof. Chris Barnes<br>christopher.barnes@ucl.ac.uk</h2>
				<p>I'm a Professor in the Division of Biosciences within the Faculty of Life Sciences at
					University College London. My career has spanned multiple fields including particle
					physics, genetics, statistics, computational systems biology and synthetic biology (<a href=https://scholar.google.co.uk/citations?user=aVCoSW8AAAAJ&hl=en>Google Scholar</a>). In
					addition to my research, I'm also actively engaged in training the next generation of
					researchers; I developed the BBSRC funded e-learning resource <a href=http://www.sysmic.ac.uk> SysMIC </a>
					to train life scientists in computational and mathematical skills. I'm also involved in running the
					EPSRC funded Centre for Doctoral Training in BioDesign Engineering and the LIDo Doctoral Training Partnership.</p>
				<a class="close" href="#close"></a>
			</div>
			<!--- END BLOCK ----->
			<a href="#x" class="overlay" id="Linda_Dekker"></a>
			<div class="popup">
				<h2>Dr Linda Dekker<br>linda.dekker.09@ucl.ac.uk</h2>
				<p>I am currently working on a project constructing, characterising and optimising
					whole-cell bacterial biosensors that detect metabolites important in gut microbiota
					disorders. I completed my BSc and MSc in Microbiology at the University of Otago
					(New Zealand) and did a PhD in Microbiology in Prof Joanne Santini's group at UCL. I
					previously worked as a PDRA at Imperial College London on various synthetic biology
					projects ranging from natural product biosynthesis in Clostridia, to changing the material
					properties of bacterial cellulose.  I'm passionate about Microbiology and am a Microbiology
					Society Champion and try to promote the awesome opportunities the
					<a href=https://www.microbiologyresearch.org/>Microbiology Society</a> has to offer. I
					founded the Barnes Lab cake club &#x1F9C1;.
				</p>
				<a class="close" href="#close"></a>
			</div>

			<a href="#x" class="overlay" id="#Pedro_Fontanarrosa"></a>
			<div class="popup">
				<h2>Dr Pedro Fontanarrosa<br> pfontanarrosa@gmail.com</h2>
				<p>I am a Postdoctoral Research Associate in the CSSB lab, where my primary focus is on the 
					intersection of artificial intelligence and microbiology. My academic path began at the 
					University of Buenos Aires (UBA), Argentina, where I earned both my Bachelor's and Master's 
					degrees in Evolutionary Biology. I furthered my studies in the United States, obtaining a 
					Master's in Bioengineering and a Ph.D. in Biomedical Engineering from the University of Utah. 
					My doctoral research laid the foundation for my postdoctoral fellowship at the University of Boulder, 
					Colorado, where I delved into the intricacies of genetic circuitry. Currently, at UCL, my work 
					revolves around leveraging AI and machine learning techniques to model and engineer resilient 
					microbial communities. 
				</p>
				<a class="close" href="#close"></a>
			</div>

			<a href="#x" class="overlay" id="Soutrick_Das"></a>
			<div class="popup">
				<h2>Dr Soutrick Das<br>soutrick.das@ucl.ac.uk</h2>
				<p>I am a post-doctoral research associate in the CSSB lab, Division of Biosciences at University College London. 
					My current research focuses on spatial computation in the context of synthetically engineered microbial 
					populations, incorporating artificial neural networks, machine learning and mathematical modeling techniques. 
					Previously, I completed my Ph.D. under the guidance of Prof. Debashis Barik at the School of Chemistry, 
					University of Hyderabad, India. Here,  I employed mathematical modeling tools to investigate the roles of 
					diverse functional motifs and chemical noise in biological reaction networks. Notably, we developed a novel 
					protocol to automate the modeling of canonical and noncanonical biological switches.
				</p>
				<a class="close" href="#close"></a>
			</div>
			
			<a href="#x" class="overlay" id="Kim_Owen"></a>
			<div class="popup">
				<h2>Kimberley Owen<br>k.owen20@imperial.ac.uk</h2>
				<p>I am a PhD student on the CDT for Biodesign
					Engineering. I am new to synthetic biology and
					looking forward to expanding my skill set. I
					spent the summer of 2020 working at the Milton
					Keynes Lighthouse Lab (UKBiocentre) testing
					covid-19 samples. I completed my Bachelor’s in
					Biochemistry at the University of Bath (2020)
					and conducted a year in research (2018/19) at
					the University of Arizona (Herbst-Kralovetz
					group) where I was introduced to the complex
					interplay between host and microbiota in the
					female reproductive tract. My PhD project will
					involve developing new bacterial therapeutics
					for targeting and manipulating the cancer microbiome.
				</p>
				<a class="close" href="#close"></a>
			</div>
			<a href="#x" class="overlay" id="Jurgen_Riedel"></a>
                        <div class="popup">
                                <h2>Jurgen Riedel<br>jurgen.riedel@gmail.com</h2>
                                <p>I am a PhD student at University College London in the Department of Cell and Developmental Biology. 
				   I've had a career as a Data Analyst and Scientist in many industries, including private banking, energy and biotechnology. 
				   I hold a PhD in Theoretical Physics from the University of Oldenburg, Germany, where my research focused on black holes, quantum gravity and dark matter. 
				   I'm also deeply interested in complexity theory and its application to large universe masses, including nature at the cellular level. 
				   My current research focuses on reaction-diffusion systems in the continuous domain via PDEs and the discrete domain via cellular automata. 
				   My goal is to study complex systems at the edge of chaos and observe the emergence of complex behaviour.
                                </p>
                                <a class="close" href="#close"></a>
			</div>
			<a href="#x" class="overlay" id="Chania_Clare"></a>
                        <div class="popup">
                                <h2>Chania Clare<br>chania.clare.21@ucl.ac.uk</h2>
                                <p>I’m a PhD student on the London Interdisciplinary Doctoral Programme (LIDo), where my work looks into the role of bacterial microcompartments in community dynamics though computational and wetlab techniques. I completed my Bachelor’s and Master’s degrees at Imperial College London, where I investigated electrotropism in plants, engineering cyanobacteria to support crop growth, and circadian-dependent immunity in the malaria parasite. As part of my PhD I completed a rotation project with the London School of Hygiene and Tropical Medicine and the Francis Crick Institute to identify novel drug targets for malaria parasite invasion.
                                </p>
                                <a class="close" href="#close"></a>
			</div>
			<a href="#x" class="overlay" id="Kathleen_Zhang"></a>
                        <div class="popup">
                                <h2>Kathleen Zhang<br>kathleen.zhang@ucl.ac.uk</h2>
                                <p>I am a PhD student on the EPSRC Centre for Doctoral Training in BioDesign Engineering program. My research focuses on applying analogue computing to biologics by using artificial neural network and spatial patterning of engineered bacteria. Previously, I worked as a Platform Scientist for 2 years at LabGenius, a London based biotech start-up in drug discovery at the interface of ML and Synbio. I have a Master of Science in Cell and Systems Biology from the University of Toronto, Canada, in the McMillen Synthetic Biology and Cellular Control lab where I worked on developing stress responsive biosensors in probiotic bacteria for detection of inflammation in IBD; and a Bachelor of Science in Honours Biology from the University of Waterloo.
                                </p>
                                <a class="close" href="#close"></a>
			</div>
			<a href="#x" class="overlay" id="Casey_Chen"></a>
                        <div class="popup">
                                <h2>Casey Chen<br>casey.chen.23@ucl.ac.uk</h2>
                                <p>I am a PhD student on the EPSRC CDT BioDesign Engineering programme. I am interested in engineering biological systems so they perform as we desire. 
									My current project focuses on tuning microbial interactions to establish microbial communities and using computational tools to predict and design them. 
									Before joining the group, I worked as a research assistant at the Chinese Academy of Sciences (SIAT), focusing on the characterisation of a biofilm-derived electroconductive protein. 
									I completed an MRes in Systems and Synthetic Biology at Imperial College London and BSc in Molecular Genetics at the University of Edinburgh.
                                </p>
                                <a class="close" href="#close"></a>
			</div>
		</div>
	</div>
	<div class="content dark style1">
		<div class="container">
			<h1> <b>Past group members</b></h1>
		    Dr Jack Rutter. PhD/PDRA 2017-2024. Now a Design and Policy Advisor, Department for Education <br>
			Ms Qing Ong. Research Assistant 2022-2024. Now doing a PhD with Tom Gorochowski, University of Bristol <br>
			Dr Mohamed Ali al-Badri. PDRA  2020-2024. Now a Senior ML Engineer at Nucleome Therapeutics<br>
			Dr Bingxin Lu. PDRA 2019-2023. Now a Surrey Future Fellow at <a href="https://www.surrey.ac.uk/people/bingxin-lu"> University of Surrey</a><br>
			Dr Will Cross. Visiting Fellow 2021-2023. Now a Lecturer (Assistant Prof.) University of Reading <br>
			Dr Alex Fedorec. PhD/PDRA 2014-2022. Now a Lecturer (Assistant Prof.) at <a href="https://profiles.ucl.ac.uk/41088-alexander-fedorec">UCL</a><br>
            Dr Neythen Treloar. PhD/PDRA 2018-2023. Now a Senior Data Scientist at Bactobio <br>
            Dr Ke Yan Wen. PDRA 2019-2022. Now a Senior Scientist at BactoBio <br>
		    Dr Behzad Karkaria. PhD/PDRA 2017-2021. Now a AI/ML Engineer at GSK<br>
			Dr Luca Rosa. PhD student 2017-2021. Now on the Faculty.ai programme<br>
			Ms Clare Robinson. Research Assistant 2019-2020. Now doing a PhD with David Riglar, Imperial College  <br>
			Dr Josh Russell-Buckland. PhD student 2016-2020. Now a software engineer at Guardian News and Media <br>
			Dr Marc Williams. PhD student 2014-2018. Now a Research Fellow with Sohrab Shah, Memorial Sloan Kettering <br>
			Dr Tanel Ozdemir. PhD student 2013-2017. Now an Investment Associate at AlbionVC <br>
			Dr Lourdes Sriraja. PhD student 2013-2017. Now a PDRA with Tariq Enver, UCL Cancer Institute <br>
			Dr Mae Woods. PDRA 2013-2018. Now at Baylor College of Medicine  <br>
			Dr Miriam Leon. PhD student 2012-2016. Now a Senior Data Scientist at Lyft <br>
			<br>
		</div>
	</div>
</section>

<section id="research_cancer" class="main">
	<header>
		<div class="container">
			<h2>Mutational processes</h2>
			<figure>
				<img width="50%" src="images/science/PDTO-model.jpg" alt="" />
				<figcaption>Fig: Modelling chromosomal gain and loss within a stochastic branching process </figcaption>
		  	</figure>
			<p> MA al-Badri, WCH Cross, CP Barnes (2024).
				Explainable deep learning on 7500 whole genomes elucidates cancer-specific patterns of chromosomal instability.
			  	bioRxiv, <a href=https://doi.org/10.1101/2024.03.08.584160> 2024.03.08.584160 </a></p>

			<p> B Lu, S Winnall, W Cross, CP Barnes (2024).
				Cell-cycle dependent DNA repair and replication unifies patterns of chromosome instability.
				bioRxiv, <a href=https://doi.org/10.1101/2024.01.03.574048> 2024.01.03.574048</a></p>

		  	<p> Lu, B., Curtius, K., Graham, T.A., Yang, Z., Barnes, C.P. (2023). 
				CNETML: maximum likelihood inference of phylogeny from copy number profiles of multiple samples. 
				Genome biology, 24 doi:<a href=https://genomebiology.biomedcentral.com/articles/10.1186/s13059-023-02983-0>10.1186/s13059-023-02983-0</a></p>
			
			<p> Gabbutt, C., Schenck, R.O., Weisenberger, D.J., Kimberley, C., Berner, A., Househam, J., ...Patel, R. (2022).
		  		Fluctuating methylation clocks for cell lineage tracing at high temporal resolution in human tissues. 
				Nature Biotechnology, doi:<a href=https://www.nature.com/articles/s41587-021-01109-w>10.1038/s41587-021-01109-w</a></p>
		  
		    <p> Bollen, Y., Stelloo, E., van Leenen, P., van den Bos, M., Ponsioen, B., Lu, B., ...Snippert H.J.G. (2021). 
			  	Reconstructing single-cell karyotype alterations in colorectal cancer identifies punctuated and gradual diversification patterns. 
			  	Nature Genetics, doi:<a href=https://www.nature.com/articles/s41588-021-00891-2>10.1038/s41588-021-00891-2</a>
			</p>
			
			<p> Williams, M.J., Zapata, L., Werner, B., Barnes, C.P., Sottoriva, A., Graham, T.A. (2020).
				Measuring the distribution of fitness effects in somatic evolution by combining clonal dynamics with dN/dS ratios.
				eLife, 9 doi:<a href=https://doi.org/10.7554/eLife.48714>10.7554/eLife.48714</a>
			</p>
			
			<p>Williams, M.J., Werner, B., Heide, T., Curtis, C., Barnes, C.P., Sottoriva, A., Graham, T.A. (2018).
				Quantification of subclonal selection in cancer from bulk sequencing data.
				Nature Genetics 2018 Jun;50(6):895-903. doi:<a href=https://doi.org/10.1038/s41588-018-0128-6>10.1038/s41588-018-0128-6</a> <br>
				<a href=https://www.thetimes.co.uk/edition/news/tumour-growth-can-be-predicted-by-computer-bdqp5c6ww> Article in The Times </a>
			</p>
			
			<p>
				Woods M.L., Barnes C.P. (2016).
				Mechanistic Modelling and Bayesian Inference Elucidates the Variable Dynamics of Double-Strand Break Repair.
				PLoS Comput Biol. 2016 Oct 14;12(10):e1005131. doi:<a href=https://doi.org/10.1371/journal.pcbi.1005131>10.1371/journal.pcbi.1005131</a>
			</p>
			
			<p>
				Williams, M.J., Werner, B, Barnes, C.P., Graham, T.A., & Sottoriva, A. (2016).
				Identification of neutral tumor evolution across cancer types.
				Nature Genetics, 2016 Jan 18. doi:<a href=https://doi.org/10.1038/ng.3489>10.1038/ng.3489</a>
			</p>
			
			<p>
				Chaidos, A, Barnes, C.P. et. al. (2013).
				Clinical drug resistance linked to interconvertible phenotypic and functional states of tumor-propagating cells in multiple myeloma.
				Blood. 2013 Jan 10;121(2):318-28. doi:<a href=https://doi.org/10.1182/blood-2012-06-436220>10.1182/blood-2012-06-436220</a>
			</p>
		</div>
	</header>
</section>

<section id="research_microbiome" class="main">
	<header>
		<div class="container">
			<h2>Microbiome engineering</h2>
			<figure>
				<img width="40%" src="images/science/photo-worm-v1-1200x800.jpg" alt="" />
				<img width="35%" src="images/science/spock-grn.jpg" alt="" />
				<figcaption>
					Fig: (Left) Fluorescent bacteria colonising the gut of a C. elegans nematode worm.
					(Right)	An engineered GRN in E. coli enabling bacteriocin production control through quorum sensing (SPoCK system)
				</figcaption>
			</figure>

			<p> Engaging cancer patients on their attitudes towards microbiome engineering technologies (2024)
				KA Owen, JW Rutter, C Holland, HBT Le, P Shapira, C Lewis, JM Kinross, ...
				bioRxiv, 2024.11. 25.625255, doi:<a href=https://doi.org/10.1101/2024.11.25.625255>10.1101/2024.11.25.625255</a></p>

			<p> C Clare, JW Rutter, AJH Fedorec, S Frank, CP Barnes (2024).
				Bacterial microcompartment utilisation in the human commensal Escherichia coli Nissle 1917.
				J Bacteriol 206:e00269-24.
				doi:<a href=https://doi.org/10.1128/jb.00269-24> 10.1128/jb.00269-24</a> </p>

			<p> JW Rutter, L Dekker, C Clare, ZF Slendebroek, KA Owen, JAK McDonald, AJH Fedorec, CP Barnes (2024).
				A bacteriocin expression platform for targeting pathogenic bacterial species.
				Nature Communications 15 (1), 6332, doi:<a href=https://doi.org/10.1038/s41467-024-50591-8> 10.1038/s41467-024-50591-8</a></p>

		  	<p> Karkaria, B.D., Manhart, A., Fedorec, A.J.H., Barnes, C.P. (2022).
		  		Chaos in synthetic microbial communities.
		  		PLoS Computational Biology, 18 (10), doi:10.1371/journal.pcbi.1010548
		  	</p>


		  	<p> Rutter, J.W., Dekker, L., Owen, K.A., Barnes, C.P. (2022). 
				Microbiome engineering: engineered live biotherapeutic products for treating human disease. 
				Frontiers in Bioengineering and Biotechnology, 10 doi:10.3389/fbioe.2022.1000873 </p>

		  	<p>
			  	Rutter, J.W., Dekker, L., Fedorec, A.J.H., Gonzales, D.T., Wen, K.Y., Tanner, L.E.S., ...Barnes, C.P. (2021). 
			  	Engineered acetoacetate-inducible whole-cell biosensors based on the AtoSC two-component system.
			  	Biotechnology and Bioengineering, doi:<a href=https://onlinelibrary.wiley.com/doi/10.1002/bit.27897>10.1002/bit.27897</a><br>
				<a href=https://zenodo.org/record/5078479#.YRp_JGbdvUI> Download the data here </a> <br>
			  </p>
			<p>
				Fedorec, A.J.H., Karkaria, B.D., Sulu, M., Barnes, C.P. (2021).
				Single strain control of microbial consortia.
				Nat Commun, 12, 1977. doi:<a href=https://www.nature.com/articles/s41467-021-22240-x>10.1038/s41467-021-22240-x</a><br>
				<a href=https://zenodo.org/record/4560355#.YRqAT2bdvUI> Download the data here </a> <br>
			</p>
			<p>
				Karkaria, B.D., Fedorec, A.J.H., Barnes, C.P. (2021).
				Automated design of synthetic microbial communities.
				Nat Commun, 12, 672. doi:<a href=https://www.nature.com/articles/s41467-020-20756-2>10.1038/s41467-020-20756-2</a><br>
				<a href=https://doi.org/10.5281/zenodo.4286040> Download the data here </a> <br>
				<a href=https://github.com/ucl-cssb/AutoCD>Code available on GitHub</a>
			</p>
			<p>
				Fedorec, A.J.H., Robinson, C.M., Wen, K.Y., Barnes, C.P. (2020).
				FlopR: An Open Source Software Package for Calibration and Normalization of Plate Reader and FlowCytometry Data.
				ACS Synth Biol, <a href=https://pubs.acs.org/doi/10.1021/acssynbio.0c00296>doi:10.1021/acssynbio.0c00296</a><br>
				<a href=https://doi.org/10.5281/zenodo.3977408> Download the data here </a><br>
				<a href=https://github.com/ucl-cssb/flopr>Code available on GitHub</a>
			</p>
			<p>
				Treloar, N.J., Fedorec, A.J.H., Ingalls, B., Barnes, C.P. (2020).
				Deep reinforcement learning for the control of microbial co-cultures in bioreactors.
				PLoS Comput Biol, 16 (4), e1007783. <a href=https://doi.org/doi:10.1371/journal.pcbi.1007783>doi:10.1371/journal.pcbi.1007783</a><br>
				<a href=https://zenodo.org/record/3728079#.XyAF5_jdtTY> Download the data here </a> <br>
				<a href=https://github.com/ucl-cssb/ROCC>Code available on GitHub</a>
			</p>
			<p>
				Wen, K.Y., Rutter, J.W., Barnes, C.P., Dekker, L. (2019). Fundamental Building Blocks of Whole-Cell Biosensor Design.
				In Thouand, G. (Ed.), Handbook of Cell Biosensors. (pp. 1-23). Springer International Publishing.
			</p>
			<p>
				Rutter, J.W., Ozdemir, T., Galimov, E.R., Quintaneiro, L.M., Rosa, L., Thomas, G.M., Cabreiro, F., Barnes, C.P. (2019).
				Detecting Changes in the Caenorhabditis elegans Intestinal Environment Using an Engineered Bacterial Biosensor.
				ACS Synth Biol, doi:<a href=https://doi.org/10.1021/acssynbio.9b00166>10.1021/acssynbio.9b00166 </a> <br>
				<a href=https://zenodo.org/record/3629222#.XlEPH5OgJR5> Download the data and code here </a>
			</p>
			<p>
				Shaw, L., Bassam, H., Barnes, C.P., Walker, A., Klein, N., Balloux, F. (2019).
				Modelling microbiome recovery after antibiotics using a stability landscape framework.
				ISME Journal, doi:<a href=https://doi.org/10.1038/s41396-019-0392-1>10.1038/s41396-019-0392-1 </a> <br>
				<a href=https://www.telegraph.co.uk/science/2019/03/22/single-course-antibiotics-may-cause-irreversible-damage-crucial/> Article in The Telegraph </a>
			</p>
			<p>
				Fedorec, A.J.H., Ozdemir, T., Doshi, A., Ho, Y.-.K., Rosa, L., Rutter, J., ...Barnes, C.P. (2019).
				Two New Plasmid Post-segregational Killing Mechanisms for the Implementation of Synthetic Gene Networks in Escherichia coli.
				iScience, doi:<a href=https://doi.org/10.1016/j.isci.2019.03.019>10.1016/j.isci.2019.03.019 </a> <br>
				<a href=https://data.mendeley.com/datasets/6kggw28x8m/1> Download the data and code here </a> <br>
				<a href=https://www.addgene.org/browse/article/28206875/> Plasmids available on addgene </a>

			<p> Barnes, C.P., Fedorec, A.J.H. (2018). Five amazing ways redesigning biological cells could help us fight cancer.
				<a href=https://theconversation.com/five-amazing-ways-redesigning-biological-cells-could-help-us-fight-cancer-95256> The Conversation </a>
			</p>
			<p>
				Ozdemir, T., Fedorec, A.J.H., Danino, T., Barnes, C.P. (2018).
				Synthetic Biology and Engineered Live Biotherapeutics: Toward Increasing System Complexity.
				Cell Systems, 7 (1), 5-16. doi:<a href=https://doi.org/10.1016/j.cels.2018.06.008>10.1016/j.cels.2018.06.008 </a>
			</p>
		</div>
	</header>
</section>

<section id="research_biocomp" class="main">
	<header>
		<div class="container">
			<h2>Biological computation</h2>
			<figure>
				<img width="50%" src="images/science/logic_gate_sim.jpg" alt="" />
				<figcaption>Fig: Patterning as a spatial computation. Each panel represents a different interpretation
					of a morphogen gradient produced from two sources A and B</figcaption>
			</figure>
		    
			<p>	AJH Fedorec, NJ Treloar, KY Wen, L Dekker, QH Ong, G Jurkeviciute, E Lyu, JW Rutter, KJY Zhang, L Rosa, A Zaikin, CP Barnes (2024).
				Emergent digital bio-computation through spatial diffusion and engineered bacteria
				Nature Communications 15 (1), 4896, doi:<a href="https://doi.org/10.1038/s41467-024-49264-3"> 10.1038/s41467-024-49264-3</a><p></p>
			</p>
			
			<p> Otero-Muras, I., Perez-Carrasco, R., Banga, J.R., Barnes, C.P. (2023). 
				Automated design of gene circuits with optimal mushroom-bifurcation behavior. 
				iScience, 26 (6), doi:10.1016/j.isci.2023.106836</p>

			<p> Treloar, N.J., Braniff, N., Ingalls, B., Barnes, C.P. (2022). 
				Deep reinforcement learning for optimal experimental design in biology. 
				PLoS Computational Biology, 18 (11), doi:10.1371/journal.pcbi.1010695 </p>

		    <p>Treloar, N., Wen, K.Y., Fedorec, A., Barnes, C. (2021). 
				SynBioBrain: building biological computers from bacterial populations. 
				The Project Repository Journal, doi:<a href=https://doi.org/10.54050/PRJ1117751>doi.org/10.54050/PRJ1117751</a> </p>

			<p> Karkaria, B.D., Treloar, N.J., Barnes, C.P., Fedorec, A.J.H. (2020).
				From microbial communities to distributed computing systems
				Frontiers in Bioengineering and Biotechnology 8, 834, doi:<a href=https://doi.org/10.3389/fbioe.2020.00834>doi.org/10.3389/fbioe.2020.00834</a>
			</p>
			<p>Dalchau, N., Szép, G., Hernansaiz-Ballesteros, R., Barnes, C.P., Cardelli, L., Phillips, A., Csikász-Nagy, A. (2018).
				Computing with biological switches and clocks.
				Natural Computing, 1-19. doi:<a href=https://doi.org/10.1007/s11047-018-9686-x>10.1007/s11047-018-9686-x</a>
			</p>

			<p>Perez-Carrasco, R., Barnes, C.P., Schaerli, Y., Isalan, M., Briscoe, J., Page, K.M. (2018).
				Combining a Toggle Switch and a Repressilator within the AC-DC Circuit Generates Distinct Dynamical Behaviors.
				Cell Systems. doi:<a href=https://doi.org/10.1016/j.cels.2018.02.008>10.1016/j.cels.2018.02.008</a>
			</p>

			<p>
				Boeing, P., Leon, M., Nesbeth, D.N., Finkelstein, A., Barnes, C. (2018).
				Towards an Aspect-Oriented Design and Modelling Framework for Synthetic Biology.
				Processes, 6 (9), doi:<a href=https://doi.org/10.3390/pr6090167>10.3390/pr6090167</a>
			</p>

			<p>
				Leon, M., Woods, M.L., Fedorec, A.J.H., & Barnes, C.P. (2016)
				A computational method for the investigation of multistable systems and its application to genetic switches.
				BMC Systems Biology 2016 Dec 7;10(1):130. doi:<a href=https://doi.org/10.1186/s12918-016-0375-z>10.1186/s12918-016-0375-z</a>
			</p>
			<p>
				Woods, M., Leon, M., Perez-Carrasco, R. & Barnes, C.P. (2016)
				A statistical approach reveals designs for the most robust stochastic gene oscillators.
				ACS Synthetic Biology 5 (6), pp 459–470. doi:<a href=https://doi.org/10.1021/acssynbio.5b00179>10.1021/acssynbio.5b00179</a>
			</p>
			<p>
				Cohen, M., Kicheva, A., Ribeiro, A., Blassberg, R., Page, K. M., Barnes, C. P. & Briscoe, J. (2015).
				Ptch1 and Gli regulate Shh signalling dynamics via multiple mechanisms.
				Nature Communications, 6. doi:<a href=https://doi.org/10.1038/ncomms7709>10.1038/ncomms7709</a>
			</p>
			<p>
				Cohen, M., Page, K. M., Perez-Carrasco, R., Barnes, C. P., & Briscoe, J. (2014).
				A theoretical framework for the regulation of Shh morphogen-controlled gene expression.
				Development, 141(20), 3868-3878. doi:<a href=https://doi.org/10.1242/dev.112573>10.1242/dev.112573</a>
			</p>
			<p>
				Liepe, J., Kirk, P., Filippi, S., Toni, T., Barnes, C. P., & Stumpf, M. P. H. (2014).
				A framework for parameter estimation and model selection from experimental data in systems biology using approximate Bayesian computation.
				Nature Protocols, 9(2), 439-456. doi:<a href=https://doi.org/10.1038/nprot.2014.025>10.1038/nprot.2014.025</a>
			</p>

		</div>
	</header>
</section>

<!-- Footer -->
<section id="footer">
	<ul class="icons">
		<li><a href="https://twitter.com/cssb_lab" class="icon brands fa-twitter"><span class="label">Twitter</span></a></li>
		<li><a href="https://github.com/ucl-cssb" class="icon brands fa-github"><span class="label">GitHub</span></a></li>
	</ul>
	<div class="copyright">
		<ul class="menu">
			<li>&copy; Untitled. All rights reserved.</li><li>Design: <a href="http://html5up.net">HTML5 UP</a></li>
		</ul>
	</div>
</section>

<!-- Scripts -->
<script src="assets/js/jquery.min.js"></script>
<script src="assets/js/jquery.scrolly.min.js"></script>
<script src="assets/js/browser.min.js"></script>
<script src="assets/js/breakpoints.min.js"></script>
<script src="assets/js/util.js"></script>
<script src="assets/js/main.js"></script>

</body>
</html>
